database/migrations/2018_06_04_125900_create_email_events_table.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateEmailEventsTable extends Migration
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('EMAIL_EVENT', function (Blueprint $table) {
```

Comment:
Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.